plugins/nodes/src/js/types/IWidthArgs.tsx

```diff
@@ -0,0 +1,8 @@
+// TODO: we need another release which exports this type so we can import
```

Comment:
Why not do it now?

Reply:
This PR blocks all the other Issues and this is something that we easily can fix later IMHO.

Reply:
Then let's have a Jira for this todo